### DISCUSSION TITLE
Error when importing Theme

### DISCUSSION THREAD

**Discussion by kellsbells** [twilio-labs/paste#2096] [General] [answered]
Following the manual instructions on https://paste.twilio.design/getting-started/manual-installation/
When I got to import 
`import {Theme} from '@twilio-paste/core/theme';`
and wrap my App in <Theme.Provider> I'm given the error
```
Compiled with problems:X

WARNING in ../../node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1:37729-37731

Should not import the named export 'props' (imported as 'Ie') from default-exporting module (only default export is available soon)


WARNING in ../../node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1:37797-37799

Should not import the named export 'props' (imported as 'Ie') from default-exporting module (only default export is available soon)
```
Has anyone ran into this or know why it's happening?

**Comment by SiTaggart** [comment, marked as answer]
Hi there!

This is a known issue that has come up before and we have it on our roadmap to fix.

Some context can be found in the previous thread https://github.com/twilio-labs/paste/discussions/1497.

Essentially, it's to do with Webpack v5, and we're not ready for it yet. It's not been the highest of priorities because up until now it was only a warning. It seems though that next.js and create react app are now shipping and enabling Webpack v5 as defaults, and we might need to address that sooner rather than later.

If it's possible, using Webpack v4 will resolve the warning. Otherwise, hang tight with the log spam and we'll get a fix out soon as we can.

**Reply by kellsbells** [reply]
Sounds good, and just for context it's a bit more than log spam. Every time my app rerenders I get this dismissible error
<img width="1508" alt="Screen Shot 2021-12-15 at 12 50 09 PM" src="https://user-images.githubusercontent.com/7221117/146255365-a7b8f592-b7cb-4128-ae08-7c8ddacee930.png">

**Reply by SiTaggart** [reply]
By any chance is that a Create React App? If so, if you downgrade to v4 and use webpack 4, this will go away I think.

**Reply by andioneto** [reply]
Hi @kellsbells 👋 It's been a minute, and we want to check in to see if you still need assistance with this. It looks like this was an issue related to Webpack v5, and I see that [we merged that fix in early January](https://github.com/twilio-labs/paste/pull/2107).

If all is well, we can go ahead and mark this as answered. Thank you!
